kafka-ui-react-app/src/components/common/JSONViewer/JSONViewer.tsx

```diff
@@ -1,15 +1,60 @@
 import React from 'react';
 import JSONTree from 'react-json-tree';
+import useDataSaver from 'lib/hooks/useDataSaver';
 import theme from './themes/google';
+import DynamicButton from './DynamicButton';
 
 interface JSONViewerProps {
   data: {
     [key: string]: string;
   };
 }
 
-const JSONViewer: React.FC<JSONViewerProps> = ({ data }) => (
-  <JSONTree data={data} theme={theme} shouldExpandNode={() => true} hideRoot />
-);
+const JSONViewer: React.FC<JSONViewerProps> = ({ data }) => {
+  const { copyToClipboard, saveFile } = useDataSaver();
+  const copyButtonHandler = () => {
+    copyToClipboard(JSON.stringify(data));
+  };
+  const buttonClasses = 'button is-link is-outlined is-small is-centered';
+  return (
+    <div>
+      <div
+        className="field has-addons"
+        style={{
```

Comment:
You can use Bulma helpers https://bulma.io/documentation/helpers/flexbox-helpers/#justify-content